client/my-sites/email/inbox/mailbox-selection-list/index.jsx

```diff
@@ -93,7 +96,9 @@ const NewMailboxUpsell = () => {
 				<div>{ translate( 'Create new and activate immediately' ) }</div>
```

Comment:
Not related to this PR, but I think we need to improve this text - it doesn't feel right to me. I am honestly not sure how we should push this, as "activate immediately" doesn't really feel enticing to me.

Maybe @semihakocer has some thoughts on this? (We should definitely take this on in a follow-up PR.)